tribe-ext-formatted-event-date-shortcode/index.php

```diff
@@ -13,6 +13,7 @@
 // 2016-09-15 Cliff created for https://theeventscalendar.com/support/forums/topic/_eventstartdate-format/
 // 2016-12-07 Cliff turned into an extension
 // 2016-12-21 Cliff modified to overcome bugs in TEC Core and improve this extension's code and error messages (zero downloads at time of editing)
+// 2017-02-08 version 1.1 to make output translatable (e.g. month names), add event ID and timestamp data attributes, and enable using PHP date format constants
```

Comment:
I would remove this entire block from the extension itself. We are trying to keep our changelogs within the Extensions webpage. And Git already thoroughly documents who wrote what line of code. Afaing, cutting down on redundancy will make our code easier to maintain.